detectable/src/main/java/com/synopsys/integration/detectable/detectables/bitbake/BitbakeSession.java

```diff
@@ -92,6 +117,6 @@ private ExecutableOutput runBitbake(String bitbakeCommand) throws ExecutableRunn
             sourceCommand.append(" ");
             sourceCommand.append(sourceArgument);
         }
-        return executableRunner.execute(ExecutableUtils.createFromTarget(workingDirectory, bashExecutable, "-c", sourceCommand.toString() + "; " + bitbakeCommand));
+        return executableRunner.execute(ExecutableUtils.createFromTarget(sourceDir, bashExecutable, "-c", sourceCommand + "; " + bitbakeCommand));
```

Comment:
This should be changed to use `ExecutableRunner::executeSuccessfully`. Then we don't need to do checks on the return code in `this::executeBitbakeForEnvironment` and `this:: executeBitbakeForDependencies`